device/dhcp/servers/v6/dhcpv6server.yaml

```diff
@@ -0,0 +1,199 @@
+          x-field-uid: 4
+        ianapd:
+          $ref: '#/components/schemas/Dhcpv6Server.PoolInfo'
+          x-field-uid: 5
+
+    Dhcpv6Server.PoolInfo:
+      description: >-
+        The container for pool configurations for IA types iana,iata and ianapd.
+      type: object
+      properties:
+        start_address:
+          description: >-
+            The IP address of the first lease pool.
+          type: string
+          format: ipv6
+          default: ::a0a:101
+          x-field-uid: 1
+        prefix:
+          description: >-
+            The IPv6 network prefix length to be applied to the address. 
+          type: integer
+          format: uint32
+          default: 64
+          maximum: 128
+          x-field-uid: 2
+        size:
+          description: >-
+            The total number of addresses in the pool.
+          type: integer
+          format: uint32
+          default: 10
+          minimum: 1
+          maximum: 1000000
+          x-field-uid: 3
+        step:
+          description: >-
+            The increment value for the lease address within the lease pool where multiple
+            addresses are present. The value is incremented according to the Prefix
+            Length and Step.
+          type: string
+          format: ipv6
+          default: 0::1
+          x-field-uid: 4
+
+    Dhcpv6Server.PrefixPoolInfo:
+      description: >-
+        The container for prefix pool configurations for IA type iapd.
+      type: object
+      properties:
+        start_prefix_address:
+          description: >-
+            The IP address of the first prefix pool.
+          type: string
+          format: ipv6
+          default: a1a1::0
+          x-field-uid: 1
+        prefix:
+          description: >-
+            The IPv6 network prefix length to be applied to the address. 
+          type: integer
+          format: uint32
+          default: 64
+          maximum: 128
+          x-field-uid: 2
+        size:
+          description: >-
+            The total number of addresses in the pool.
+          type: integer
+          format: uint32
+          default: 10
+          minimum: 1
+          maximum: 1000000
+          x-field-uid: 3
+        step:
+          description: >-
+            The increment value for the lease address within the lease pool where multiple
+            addresses are present. The value is incremented according to the Prefix
+            Length and Step.
+          type: string
+          format: ipv6
+          default: 0:0:0:1:0:0:0:0
+          x-field-uid: 4
+    
+    DhcpV6Server.Advanced:
+      description: >-
+        Optional configuration for DHCPv6 server.
+      type: object
+      properties:
+        primary_dns_server:
+          description: >-
+             The primary DNS server address that is offered to DHCP clients that request this information through a TLV option.
+          type: string
+          format: ipv4
```

Comment:
There is no default for this . So what should happen if this is not configured . Will it return :: ? 
If this object is explicitly configured , should these be required since a default might not be making sense ?

Reply:
added primary as required  and added  advanced dns object for secondary